src/porepy/numerics/fv/fvutils.py

```diff
@@ -584,6 +584,7 @@ def map_hf_2_f(fno=None, subfno=None, nd=None, g=None):
     ).tocsr()
     return hf2f
 
+
```

Comment:
Does black do this? 

Codacy had quite a few complaints on trailing whitespaces etc.